docs/pages/foundations/iconography/usage.js

```diff
@@ -518,50 +542,60 @@ Generally 16px and 24px icons should be used in mobile interfaces. When icons ar
             cardSize="md"
             type="do"
             description="Style: Use our icons following the Gestalt design specs."
-            defaultCode={`
-<Icon icon="pincode" accessibilityLabel="pin code" color="default" size={32} />
-`}
+            sandpackExample={
+              <SandpackExample
+                name="Use icons following design specs"
+                code={useIconsFollowingDesignSpecs}
+                hideEditor
+                hideControls
+                layout="column"
+              />
+            }
           />
           <MainSection.Card
             cardSize="md"
             type="don't"
             description="Modify our icon's design specs. Changing an icon shape can make the icon hard to read for Pinners and feel like a different visual language."
-          >
-            <OutlinedPincode />
-          </MainSection.Card>
+            sandpackExample={
+              <SandpackExample
+                name="Don't modify our icon's design specs"
+                code={dontModifyIconDesignSpecs}
+                hideEditor
+                hideControls
+                layout="column"
+              />
+            }
+          />
         </MainSection.Subsection>
         <MainSection.Subsection columns={2}>
           <MainSection.Card
             cardSize="md"
             type="do"
             description="Alignment: Center-align icons when they’re next to a text element."
-          >
-            <Flex
-              gap={{
-                row: 1,
-                column: 0,
-              }}
-              alignItems="center"
-            >
-              <ReactionLove />
-              <Text>2.8k</Text>
-            </Flex>
```

Comment:
@AlbertCarreras @rlingineni Do you think it would be a good idea to keep "inline" examples (instead of putting them inside Sandpack)?